.pipelines/azure-pipeline-build.yml

```diff
@@ -404,6 +404,9 @@ stages:
             --artifact-type 'application/vnd.cncf.notary.signature' \
             ./payload.json:application/cose \
             -a "io.cncf.notary.x509chain.thumbprint#S256=[\"79E6A702361E1F60DAA84AEEC4CBF6F6420DE6BA\"]"
+          oras attach $(LINUX_FULL_IMAGE_NAME) \
+            --artifact-type 'application/vnd.microsoft.artifact.lifecycle' \
+            --annotation "vnd.microsoft.artifact.lifecycle.end-of-life.date=$(date -u -d '+1 hour' +"%Y-%m-%dT%H:%M:%SZ")"
```

Comment:
Any reason why we are adding 1 hour ? We can just use current time , or start of the day time (00:00:00). It also allows past time as well, so that means {currentTime -1} hour would work even better.

Reply:
Thanks @vishiy ! I was thinking adding 1 hour would account for some buffer time after the build step succeeds but I agree it will be better to do past time (1 hour) instead so that our images are not flagged. Making this change now.

Reply:
Updated the annotation. Thanks!